<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>431</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>431</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>431</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/dinosaur.jpeg</normaloff>../../../Downloads/dinosaur.jpeg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: rgb(43, 43, 43)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 55, 252);
color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 55, 252)</string>
    </property>
    <property name="text">
     <string>Agregar Invitado</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>Apellido</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSurname">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>Sexo</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255)</string>
    </property>
    <item>
     <property name="text">
      <string>Hombre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mujer</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtDni">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>Dni</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtSurname</tabstop>
  <tabstop>txtDni</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addGuest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addGuest()</slot>
 </slots>
</ui>
